config/initializers/riiif.rb

```diff
@@ -19,5 +19,5 @@
 
   # Riiif.not_found_image = 'app/assets/images/us_404.svg'
   #
-  Riiif::Engine.config.cache_duration = 365.days
+  Riiif::Engine.config.cache_duration = 30.days
```

Comment:
I think this is server side caching, sorry that makes more sense so 30 days sounds better!